samples-generator/bin/3d-tiles-samples-generator.js

```diff
@@ -277,12 +284,9 @@ var promises = [
     createExpireTileset()
 ];
 
-Promise.all(promises)
-    .then(function() {
-        fsExtra.remove('C:/Code/3d-tiles-samples/localTilesets/output');
-        fsExtra.copy(outputDirectory, 'C:/Code/3d-tiles-samples/localTilesets/output');
-        console.log('Done');
-    });
+return fsExtra.emptyDir(outputDirectory).then(function () {
+    return Promise.all(promises);
+});
```

Comment:
Tweak the promise style slightly:
```js
return fsExtra.emptyDir(outputDirectory).then(function () {
    return Promise.all(promises);
});
```

```js
return fsExtra.emptyDir(outputDirectory)
    .then(function() {
        return Promise.all(promises);
    });
```

Also thanks for removing the testing code I forgot about in this branch.